app/common/data/migrations/versions/007_add_grant_to_org_fk.py

```diff
@@ -0,0 +1,44 @@
+"""track ownership of grants by orgs (eg MHCLG)
+
+Revision ID: 007_add_grant_to_org_fk
+Revises: 006_add_another
+Create Date: 2025-10-15 18:32:12.661381
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "007_add_grant_to_org_fk"
+down_revision = "006_add_another"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    with op.batch_alter_table("grant", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("organisation_id", sa.Uuid(), nullable=True))
```

Comment:
I'm guessing this will just be for a future tickets A/C (along with the dev and design work you mention in commits for how this is set up) to make this required in the future